test/conformance/cf_header.md

```diff
@@ -0,0 +1,39 @@
+# Kubernetes Conformance Test Suite - v1.7
```

Comment:
It's 1.10 now

Reply:
We will have to refresh once new conformance tests added to 1.10 right?

Reply:
I have updated the document to 1.9.